howtos/introduction-of-a-new-tc-member.md

```diff
@@ -2,13 +2,24 @@
 
 Checklist for introducing a new Technical Committee member
 
-- Become an owner of Eiffel Community GitHub organization
+## GitHub
+- Become an [owner of Eiffel Community GitHub organization](https://github.com/orgs/eiffel-community/people?query=role%3Aowner)
 - Become a maintainer of eiffel-community-maintainers in the [maintainers list](https://github.com/orgs/eiffel-community/teams/eiffel-community-maintainers/members)
-- Become an owner of Eiffel Community Google Groups in the [members list](https://groups.google.com/g/eiffel-community/members)
-- Become an owner of Eiffel Technical Committee Google Groups in the [members list](https://groups.google.com/g/eiffel-tc/members)
+
+## Google Groups
+- Become an owner of Eiffel Community Google Group in the [members list](https://groups.google.com/g/eiffel-community/members)
+- Become an owner of Eiffel Technical Committee Google Group in the [members list](https://groups.google.com/g/eiffel-tc/members)
+- Become an owner (?) of each maintainer Google Group in the [members list](https://groups.google.com/g/eiffel-.../members)

```

Comment:
This needs to be sorted out and updated before merging this PR

Reply:
It was decided on the last TC meeting that we should not continue using Google groups for the maintainers. At least not demanded from TC and not tightly connected to the GitHub repos. Updates to PROJECTS.md will happen through another issue/PR but we remove this statement here now.